Client/Frontend/Home/FeedbackCard.swift

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO: Remove before going to production
+
+import SnapKit
+import Storage
+import Shared
+import Core
+
+final class FeedbackCard: UIView {
+    public var dismissClosure: (() -> Void)?
+    lazy var title: UILabel = {
+        let title = UILabel()
+        title.text = .localized(.helpUsImprove)
+        title.numberOfLines = 0
+        title.lineBreakMode = .byWordWrapping
+        title.font = UIFont.systemFont(ofSize: 20, weight: .bold)
```

Comment:
Usually I'd say let's stick to the way FF does dynamic fonts, but as it's only for a short living feature it's ok to leave it :)

Reply:
I didn't change fonts here, I copy/pasted what it is being used on their own promo.